torchtext/__init__.py

```diff
@@ -4,7 +4,7 @@
 from . import vocab
 from . import legacy
 
-__version__ = '0.4.0'
+__version__ = '0.5.0'
```

Comment:
Please never bump the version randomly without doing a release or outside of a release branch.

Reply:
I thought after each release, we bump the version up for next release.

Reply:
Fixed.